server/server.go

```diff
@@ -245,6 +315,9 @@ func (s *Server) Start() error {
 	handler.Handle("/messages/images", &imageHandler{db: s.db, logger: s.logger})
 	handler.Handle("/messages/audio", &audioHandler{db: s.db, logger: s.logger})
 	handler.Handle("/messages/identicons", &identiconHandler{logger: s.logger})
+	handler.Handle("/messages/sticker", &stickerHandler{db: s.db, logger: s.logger, downloader: s.downloader})
+	handler.Handle("/ipfs", &ipfsHandler{logger: s.logger, downloader: s.downloader})
```

Comment:
how will ipfs be used?

Reply:
http://localhost:1234/ipfs/hash=sticker_hash_goes_here

Reply:
i mean, will it be used from client somehow ?

Reply:
i don't see where we can use it in client,just wondering why do wee need it

Reply:
It's being used in `stickers_market` in the preview and thumbnail attributes. I'm going to remove  /messages/ipfs since as it was pointed out in other comments, we could access the hash directly so there's no need of using the messageId